index.js

```diff
@@ -2,13 +2,28 @@ const express = require("express");
 const port = process.env.PORT || 3000;
 const app = express();
 const addBookRouter = require('../server/routes/addBook')
-const volleyball = require('volleyball')
+const morgan = require('morgan')
 const cors = require('cors')
+const dotenv = require('dotenv')
+
+dotenv.config()
 
 app.use(cors());
 app.use(express.json())
 app.use(addBookRouter)
-app.use(volleyball)
+app.use(morgan('dev'))
+
+const mongoose = require("mongoose");
+mongoose.connect(process.env.DATABASE_URL, {
+  useNewUrlParser:true,
+  useUnifiedTopology: true
+})
+
+const db = mongoose.connection
+
+db.on('error', error => console.log(error))
+db.once('open', () => {console.log('Database connected');
+})
 
 
 app.listen(port, () => {
```

Comment:
You should add a 404 and error middleware here.

See an example here:
https://github.com/w3cj/express-api-starter/blob/master/src/app.js#L26
https://github.com/w3cj/express-api-starter/blob/master/src/middlewares.js